examples/7nodes/init.sh

```diff
@@ -71,7 +57,7 @@ if  [[ "$numNodes" -gt 7 ]] ; then
 fi
 
 # check if the number of nodes is less than 7. if yes dynamically create the permissioned-nodes.json
-createPermissionedNodesJson $numNodes
+./create-permissioned-nodes.sh $numNodes
```

Comment:
I don't think you need the call to `create-permissioned-nodes.sh` here , as `init.sh` always calls the relevant `$consensus-init.sh` (which will call `create-permissioned-nodes.sh`).

Reply:
Agree. changes done.